character-controller.js

```diff
@@ -939,6 +939,22 @@ class InterpolatedPlayer extends StatePlayer {
       quaternion: this.quaternionInterpolant.get(),
     };
   }
+  update(timestamp, timeDiff) {
+    this.updateInterpolation(timeDiff);
+
+    const mirrors = metaversefile.getMirrors();
+    if(!this.avatar) return console.warn("no avatar");

```

Comment:
`throw`?

Reply:
i will test, if its legit then we will remove the warning